Packs/Akamai_WAF/Integrations/Akamai_WAF/Akamai_WAF.yml

```diff
@@ -1224,7 +1233,13 @@ script:
     outputs:
     - contextPath: Akamai.PapiProperty.DefaultRule
       description: Papi Property default rule
-  dockerimage: demisto/akamai:1.0.0.39599
+  - arguments:
+    - description: The path that includes enrollmentId and changeId
+      name: change_path
+      required: true
+    description: acknowledge pre verification warning
```

Comment:
I don't understand this description.  Please provide more information.

Reply:
This command is to acknowledge the pre-verification warning message after initiate an enrollment change. When there is a conflict between the existing certificate and the new certificate, there will be a warning message popped out, which needs to be acknowledged either manually or automatcally by API Call. This command is the one to acknowledge the warning. After it is acknowledged, the status of the enrollment will move to the next phase.